### PR TITLE
docs: complete sentence documenting turning on parserOptions.project 

### LINKED ISSUE #6992: Docs: what parserOptions.project relative to?
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

https://github.com/typescript-eslint/typescript-eslint/blob/main/docs/linting/Typed_Linting.mdx

```
## Specifying TSConfigs

The `parserOptions.project` option can be turned on with either:

- `true`: to always use `tsconfig.json`s nearest to source files
- `string | string[]`: any number of glob paths to match TSConfig files relative to the

For example, if you use a specific `tsconfig.eslint.json` for linting, you'd specify:
```

Second item ends with "relative to the" without noun.  It seems to be "config file" but I'm not sure.

### Affected URL(s)

https://typescript-eslint.io/linting/typed-linting#specifying-tsconfigs

### PR DESCRIPTION
**Pull request by Melandra**
## PR Checklist

- [x] Addresses an existing open issue: fixes #6992 
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

Completes sentence in https://github.com/typescript-eslint/typescript-eslint/blob/main/docs/linting/Typed_Linting.mdx documentation on how to turn parserOptions.project on with the suggestion from https://github.com/typescript-eslint/typescript-eslint/issues/6992.

### CODE REVIEW

File: docs/linting/Typed_Linting.mdx [hunk 1]
```diff
@@ -47,7 +47,7 @@ You may see new rules reporting errors based on type information!
 The `parserOptions.project` option can be turned on with either:
 
 - `true`: to always use `tsconfig.json`s nearest to source files
-- `string | string[]`: any number of glob paths to match TSConfig files relative to the
+- `string | string[]`: any number of glob paths to match TSConfig files relative to `parserOptions.tsconfigRootDir`, or the CWD if that is not provided
```
Comment (JoshuaKGoldberg) [inline docs/linting/Typed_Linting.mdx:50]: > CWD

[Suggestion] My only concern here is that some users won't know what _"CWD"_ refers to. What do you think about explicitly saying it?

```suggestion
- `string | string[]`: any number of glob paths to match TSConfig files relative to `parserOptions.tsconfigRootDir`, or the current working directory if that is not provided
```

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Thanks for this! Just one non-blocking suggestion, what do you think? 🙏

File: docs/linting/Typed_Linting.mdx (hunk 1 above)
Comment (Melandra) [reply docs/linting/Typed_Linting.mdx]: Thanks for the review! Yes, sure that sounds good to me, please see [7cb4d25](https://github.com/typescript-eslint/typescript-eslint/pull/7052/commits/7cb4d2590ef3de758834c34b4e96bade01ba6234) 🙂

**Review by JoshuaKGoldberg — APPROVED** [review]
Cute cartoon bunny clapping hands in joy

**Pull request merged**